WARN: RE2 not usable, falling back to RegExp

Starting with version `35.31.0` a warning pops up in the log:

```
 WARN: RE2 not usable, falling back to RegExp
       "err": {
         "code": "MODULE_NOT_FOUND",
         "requireStack": [
           "/usr/local/lib/node_modules/renovate/node_modules/re2/re2.js",
           "/usr/local/lib/node_modules/renovate/dist/expose.cjs",
           "/usr/local/lib/node_modules/renovate/dist/instrumentation/index.js",
           "/usr/local/lib/node_modules/renovate/dist/renovate.js"
         ],
         "message": "Cannot find module './build/Release/re2'\nRequire stack:\n- /usr/local/lib/node_modules/renovate/node_modules/re2/re2.js\n- /usr/local/lib/node_modules/renovate/dist/expose.cjs\n- /usr/local/lib/node_modules/renovate/dist/instrumentation/index.js\n- /usr/local/lib/node_modules/renovate/dist/renovate.js",
         "stack": "Error: Cannot find module './build/Release/re2'\nRequire stack:\n- /usr/local/lib/node_modules/renovate/node_modules/re2/re2.js\n- /usr/local/lib/node_modules/renovate/dist/expose.cjs\n- /usr/local/lib/node_modules/renovate/dist/instrumentation/index.js\n- /usr/local/lib/node_modules/renovate/dist/renovate.js\n    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)\n    at Function.Module._load (node:internal/modules/cjs/loader:920:27)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)\n    at require (node:internal/modules/cjs/helpers:110:18)\n    at Object.<anonymous> (/usr/local/lib/node_modules/renovate/node_modules/re2/re2.js:3:13)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:958:12)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)\n    at require (node:internal/modules/cjs/helpers:110:18)\n    at re2 (/usr/local/lib/node_modules/renovate/lib/expose.cjs:14:10)\n    at Object.<anonymous> (/usr/local/lib/node_modules/renovate/lib/util/regex.ts:12:18)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:958:12)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)\n    at require (node:internal/modules/cjs/helpers:110:18)\n    at Object.<anonymous> (/usr/local/lib/node_modules/renovate/lib/util/url.ts:6:1)"
       }
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -24,6 +24,7 @@ RUN apt-get update -y && \
 # renovate: datasource=npm depName=renovate
 ENV RENOVATE_VERSION=35.31.0
 
+# We need to run scripts here to have RE2 installed
 RUN npm install -g renovate@${RENOVATE_VERSION} && \
   npm cache clean --force && \
   # Smoke test

--- a/Dockerfile
+++ b/Dockerfile
@@ -24,7 +24,7 @@ RUN apt-get update -y && \
 # renovate: datasource=npm depName=renovate
 ENV RENOVATE_VERSION=35.31.0
 
-RUN npm install -g renovate@${RENOVATE_VERSION} --ignore-scripts && \
+RUN npm install -g renovate@${RENOVATE_VERSION} && \
   npm cache clean --force && \
   # Smoke test
   renovate --version

